bycycle/group/epochs.py

```diff
@@ -0,0 +1,93 @@
+"""Functions to compute features across 2 dimensional arrays of data."""
+
+from functools import partial
+from multiprocessing import Pool, cpu_count
+
+from bycycle.features import compute_features
+from bycycle.group.utils import progress_bar
+
+###################################################################################################
+###################################################################################################
+
+def compute_features_2d(sigs, fs, f_range, compute_features_kwargs=None,
+                        return_samples=True, n_cpus=-1, progress=None):
+    """Compute shape and burst features for a 2 dimensional array of signals.
+
+    Parameters
+    ----------
+    sigs : 2d array
+        Voltage time series, with shape [n_signals, n_points].
+    fs : float
+        Sampling rate, in Hz.
+    f_range : tuple of (float, float)
+        Frequency range for narrowband signal of interest, in Hz.
+    compute_features_kwargs : dict or list of dict
+        Keyword arguments used in :func:`~.compute_features`.
+    return_samples : bool, optional, default: True
+        Whether to return a dataframe of cyclepoint sample indices.
+    n_cpus : int, optional, default: -1
+        The number of jobs, one per cpu, to compute features in parallel.
+    progress: str, optional, default: None
```

Comment:
This doesn't take any string, but a set of distinct options, the convention for which is to list them out. 
I think it should be listed as:
{None, 'tqdm', 'tqdm.notebook'} (or similar).